orion/controllers/achieve/achieveController.go

```diff
@@ -0,0 +1,111 @@
+package achieve
+
+import (
+	"errors"
+	"net/http"
+	"strconv"
+
+	"github.com/gin-gonic/gin"
+)
+
+func GetAchievements(c *gin.Context) {
+	// Query Repo
+	achieveList := GetAllAchievements()
+
+	// JSON Response
+	c.JSON(http.StatusOK, achieveList)
+	return
+}
+
+func GetAchievement(c *gin.Context) {
+	id := StringToUint(c)
```

Comment:
I think a more descriptive function name is better here.
```
id := parseParamId(c)
```

Since this is a private function, you can make it lowercase.
Upper case functions indicate that this function will be used by other packages.